File: GAPInsurance/Gap.WepApi/Models/TypeRiskModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Gap.WepApi.Models
{
    public class TypeRiskModel
    {
        public Guid Id { get; set; }
        public string RiskName { get; set; }
        public string MaxCovering { get; set; }
    }
}

File: GAPInsurance/GapInsuranceClient/GapClientApp/src/app/shared/typerisk.model.ts
export class Typerisk {
    Id :string;
    RiskName:string;
    MaxCovering:string;
    constructor(
        _id:string,
        _riskName:string,
        _maxCovering:string

    ){
        this.Id = _id;
        this.RiskName = _riskName;
        this.MaxCovering = _maxCovering;
    }
 
}

